docs/docs/general/08-Terminology.md

```diff
@@ -0,0 +1,20 @@
+| Term                   | Meaning     |
+| ---------------------- | ----------- |
+| annotation             | Marked area around an element with metadata name, text, and bounding box coordinates. |
+| access token           | Gives you access to the askui inference in combination with your `workspace id`. Every `access token` has an expiry date and is assigned to exactly one workspace.  |
+| automation             | A set of instructions to achieve a task. |
+| bounding box           | Rectangle described by coordinates that describe an element's location. Displayed visually as red rectangle. |
+| command                | An action on the user interface. For example mouse left click or keypress. |
+| credentials            | Combination of `workspace id` and `access token` living in the same workspace. |
+| element                | Or user interface element. Building blocks websites or apps are build from. |
+| filters                | Type of an element. For example _button_. |
```

Comment:
```suggestion
| filters                | Screen for a type of an element. For example _button_. |
```

Reply:
I see why you made the change but I do not understand what **Screen** means here 😄 . @thinh-askui would you explain, please?

Reply:
Sorry, should be *screens. First I wanted to write "Filters for a type of an element."
Idea was just to use a synonym for filter. Maybe it's not the best synonym :D

Reply:
We can go with **screens**.